internal/oci/oci_test.go.orig

```diff
@@ -0,0 +1,356 @@
+package oci_test
```

Comment:
this seems unintentional

Reply:
@haircommander, good catch!

Sorry about this!

Reply:
Done.

Completed in https://github.com/cri-o/cri-o/commit/4d9585de0d2643c789a7c19ef2fbe66af80d166b.